### DISCUSSION TITLE
Arrays in parameter

### DISCUSSION THREAD

**Discussion by itmurkel** [appy-one/acebase#176] [Q&A]
Hello,
i am testing acebase in node.js and i get error messages when i work with arrays as parameter.
Consider the following code:
```js
let test = await db.ref("test").get({exclude:["series"]})
```
drop `TypeError: options.exclude must be an array`

```js
let ref = await db.query('test/series')
    .filter('timestamp', 'between', [1333245600000 - 240*60*1000, 1333245600000])
    .get()
```
drop `Error: between filter for timestamp must supply an Array compare argument containing 2 values`

**Comment by appy-one** [comment]
I'm not able to reproduce this. What version of acebase are you using?
